common/src/main/java/io/netty/util/ResourceLeakDetector.java

```diff
@@ -109,7 +109,7 @@ static Level parseLevel(String levelStr) {
         Level level = Level.parseLevel(levelStr);
 
         MAX_RECORDS = SystemPropertyUtil.getInt(PROP_MAX_RECORDS, DEFAULT_MAX_RECORDS);
-
+        MAX_RECORDS_SAMPLED = MAX_RECORDS * 100;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Cast one of the operands of this multiplication operation to a "long". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS2184)